support-frontend/assets/pages/supporter-plus-landing/twoStepPages/threeTierLanding.tsx

```diff
@@ -138,12 +136,6 @@ const paymentFrequencyButtonsCss = css`
 	}
 `;
 
-const tabletLineBreak = css`
```

Comment:
The css here prevents the line break from appearing on desktop.
If we want to reproduce this then we'll need separate config for mobile+tablet vs desktop.
We should consider how important this feature is - perhaps we can delay making this choice for now and just always have the line break in the heading.